modules/datetime.js

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line
+import { DateTime } from 'https://moment.github.io/luxon/es6/luxon.min.js';
+
+const showdate = document.querySelector('#showdate');
+
+setInterval(() => {
+  const now = DateTime.now();
+  showdate.innerHTML = now.toFormat('ccc LLL dd yyyy, hh:mm:ss a');
+}, 1000);
```

Comment:
- [ ] Great job importing Luxon to your project :clap:. However, while this is perfectly okay, it currently does not satisfy one core requirement of the project. You are required to **Install Luxon with npm**. This means when I go to your dependencies in your package.json, I should see luxon listed there. Currently this is not the case. Kindly follow through with [this resource](https://moment.github.io/luxon/#/install?id=es6) provided in the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md) to install Luxon and import it as a module.

**Example of a package.json file with the luxon dependency listed:**

![image](https://user-images.githubusercontent.com/86318284/220527728-71713e95-6507-436f-9650-b06cb30feebc.png)

**Requirement missed from the project requirements:**

![image](https://user-images.githubusercontent.com/86318284/220527846-b31f484d-fb6b-45aa-8a69-d9ce8736685f.png)